### ISSUE TITLE
Drive TORCS car with MyDriver

### ISSUE THREAD

**Issue by jerbaroo** [boing102/CI-Project#4]
The Keras NN quickly gets high accuracy on training data but we still need the NN to predict next `Command` in `MyDriver`. Trying this in `keras-0.py` but just having some dimensionality issue, shouldn't be a big problem.
